src/Resources/views/form_theme.html.twig

```diff
@@ -1,7 +1,11 @@
 {% block zicht_url_widget %}
+    <script src="{{ asset('bundles/zichturl/javascript/jquery-ui.autocomplete.min.js') }}"></script>
+    <link rel="stylesheet" href="{{ asset('bundles/zichturl/css/jquery-ui.autocomplete.min.css') }}" type="text/css"/>
+    <link rel="stylesheet" href="{{ asset('bundles/zichturl/css/jquery-ui.theme.min.css') }}" type="text/css"/>
```

Comment:
We zouden deze includes ook in de Cookiecutter kunnen opnemen in `templates/bundles/SonataAdminBundle/standard_layout.html.twig`

Reply:
Denk dat het handiger is om hier te doen. Ze worden ook in deze bundle gebruikt. Stel dat deze bundle over gaat op iets anders, dan zit er nog steeds (onnodig?) jQuery in het project...

Reply:
Ik vermoed dat we hetzelfde moeten doen voor deze https://github.com/zicht/admin-bundle/blob/release/8.x/src/Resources/public/javascript/autocomplete.js#L18

Reply:
Dan hebben we het er 2x in zitten 🤔

Reply:
Ik zou het liefst deze UrlType een AutocompleteType (uit admin-bundle) laten worden via "getParent()" en dan via de options de juiste werking afdwingen. Maar daarvoor moet AutocompleteType iets worden aangepast denk ik. Maar dat is wellicht een nog beter idee.

Reply:
Dat is wel een heel goed idee ja 👍🏻

Reply:
Dat heeft wel deze toevoeging in SF5 projecten nodig in `zicht_admin.yaml`

```yaml
zicht_admin:
    quicklist:
    ....
        Zicht\Bundle\UrlBundle\Entity\UrlAlias:
            repository: 'Zicht\Bundle\UrlBundle\Entity\UrlAlias'
            fields: ['public_url']
```

Het zou wel cool zijn als we dit met een recipe konden injecteren :slightly_smiling_face: